chains/orchestrator-relays/runtime/dancelight/src/tests/democracy_tracks.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+#![cfg(test)]
+
+use {
+    crate::{
+        tests::common::*, ConvictionVoting, OpenTechCommitteeCollective, Referenda, RuntimeCall,
+    },
+    alloc::vec,
+    frame_support::{
+        assert_ok,
+        traits::{schedule::DispatchTime, Bounded},
+        weights::Weight,
+    },
+    pallet_referenda::TracksInfo,
+    parity_scale_codec::Encode,
+    sp_core::H256,
+    sp_runtime::traits::{BlakeTwo256, Hash},
+};
+
+#[test]
+fn test_root_track_executes_with_root_origin() {
```

Comment:
Do we need a test for not being able to execute a root track without root origin?